presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/cache/CachingHiveMetastoreConfig.java

```diff
@@ -20,12 +20,13 @@
 import javax.validation.constraints.Min;
 import javax.validation.constraints.NotNull;
 
+import java.util.Optional;
 import java.util.concurrent.TimeUnit;
 
 public class CachingHiveMetastoreConfig
 {
     private Duration metastoreCacheTtl = new Duration(0, TimeUnit.SECONDS);
-    private Duration metastoreRefreshInterval = new Duration(Integer.MAX_VALUE, TimeUnit.DAYS);
```

Comment:
It's kind of odd now that `metastoreRefreshInterval` is optional while `metastoreCacheTtl` isnt

In `CachingHiveMetastore` both arguments are optional

Reply:
`metastoreCacheTtl=0` means "no caching", while `optional empty` means "cache indefinitely".